From a9a38f9f6014fd0b8c6e1779793da74c88b85b10 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 12 Aug 2015 15:11:17 +0200
Subject: [PATCH] dracut.cmdline.7.asc: document rd.iscsi.waitnet and
 rd.iscsi.testroute

(cherry picked from commit 69171aa8fa4bdd196709318582d73076677bd16c)
---
 dracut.cmdline.7.asc | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/dracut.cmdline.7.asc b/dracut.cmdline.7.asc
index 5f3dead1..1cf962e6 100644
--- a/dracut.cmdline.7.asc
+++ b/dracut.cmdline.7.asc
@@ -704,6 +704,12 @@ iscsistart -b --param node.session.timeo.replacement_timeout=30
 **rd.iscsi.ibft** **rd.iscsi.ibft=1**:
     Turn on iBFT autoconfiguration for the interfaces
 
+**rd.iscsi.waitnet=0**:
+    Turn off waiting for all interfaces to be up before trying to login to the iSCSI targets.
+
+**rd.iscsi.testroute=0**:
+    Turn off checking, if the route to the iSCSI target IP is possible before trying to login.
+
 FCoE
 ~~~~
 **fcoe=**__<edd|interface|MAC>__:__{dcb|nodcb}__::
